app/src/main/res/layout/movie_list_fragment.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@ifeins from what I saw `FrameLayout` is there to stack views on top of each other, so it doesn't sound better fitting than `LinearLayout`